src/main/java/io/kestra/plugin/googleworkspace/drive/Export.java

```diff
@@ -73,6 +73,10 @@ public Output run(RunContext runContext) throws Exception {
 
             runContext.metric(Counter.of("size", file.size()));
 
+            // For Google apps files such as doc, spreadsheet, csv - file.getSize() will return: null, 1 or 1024 (Which is incorrect value)
+            // Hardcoding file size from tempFile size in bytes that have been exported/created
+            file.setSize(tempFile.length());
+
```

Comment:
Can you move that before the metric just above so the metric correctly use the right size?

Reply:
Metric uses different size anyway (```size()``` return size of map as File extends of AbstractMap. ```getSize()``` returns size of file). I did want to change that, but wasn't sure if that was intended)

Reply:
OK, let's keep this one as is then.